app/_components/About.tsx

```diff
@@ -0,0 +1,47 @@
+import { Typography } from "@mui/material";
+import Image from "next/image";
+const About = () => {
+  return (
+    <section className="py-12 sm:px-6 lg:px-8 bg-[#fdf8f5]">
+      <div className="max-w-7xl mx-auto flex flex-col gap-8 bg-[#fdf8f5]">
+        <div className="flex items-center gap-5">
```

Comment:
```suggestion
    <Box component='section' className="py-12 sm:px-6 lg:px-8 bg-[#fdf8f5]">
      <Box className="max-w-7xl mx-auto flex flex-col gap-8 bg-[#fdf8f5]">
        <Box className="flex items-center gap-5">
```
نتجنت استخدام الhtml tags مباشرة